src/builder/Actions.sol

```diff
@@ -84,12 +86,8 @@ library Actions {
             nonce: accountState.quarkNextNonce,
             scriptAddress: CodeJarHelper.getCodeAddress(bridge.originChainId, scriptSources[0]),
             scriptCalldata: CCTP.encodeBridgeUSDC(
-                bridge.originChainId,
-                bridge.destinationChainId,
-                bridge.amount,
-                bridge.recipient,
-                originUSDCPositions.asset
-            ),
+                bridge.originChainId, bridge.destinationChainId, bridge.amount, bridge.recipient, originUSDCPositions.asset
+                ),
             scriptSources: scriptSources,
             expiry: 99999999999 // TODO: handle expiry
```

Comment:
Is this one left out on purpose?

Reply:
updated to use similar logic.

we're not testing this bridge action yet, yeah?

Reply:
Nope we're not. There's still some work left to do on the implementation for bridge action.